### PR TITLE
Update to all examples.

### PR DESCRIPTION
**Pull request by okbel**
## Update to all examples

Bumps all safe versions in every example/solution. Next on our todo is running an script weekly to automatize this process.  

### Upgrade Tailwind Config to v3 
Tailwind introduces breaking changes in the v3. All examples have the following replacements done.

1. Removing `darkMode` -> `darkMode.*$`
2. Removing the `variants` configuration -> `variants:((.|\n)*)(\}\,)`
3. Removing and/or replacing `purge` and configuring content sources: `purge:((.|\n)*?)(\]\,)`
4. Updating colors -> just one.
5. Updating outline config (none)
6. Removing unneeded classNames because of Automatic transforms and filters

### CODE REVIEW

File: edge-functions/ab-testing-simple/package.json [hunk 1]
```diff
@@ -10,16 +10,16 @@
...
-    "@types/react": "17.0.8",
-    "autoprefixer": "^10.3.7",
-    "postcss": "^8.3.11",
-    "tailwindcss": "^2.2.17",
-    "typescript": "4.3.2"
+    "@types/js-cookie": "^3.0.1",
+    "@types/react": "17.0.38",
+    "autoprefixer": "^10.4.2",
+    "postcss": "^8.4.5",
+    "tailwindcss": "^3.0.16",
```
Comment (leerob) [inline edge-functions/ab-testing-simple/package.json:22]: There was a breaking change for Tailwind, which requires some changes: https://tailwindcss.com/docs/upgrade-guide

**Comment by leerob** [comment]
Should we close all the dependabot PRs for this one?

File: edge-functions/ab-testing-simple/package.json (hunk 1 above)
Comment (okbel) [reply edge-functions/ab-testing-simple/package.json:22]: Done! Thanks for pointing it out!

**Comment by leerob** [comment]
FYI, I closed all of the Dependabot PRs for updating Next.js to latest `12.0.9` - could we do that here? 🙏

**Review by lfades — APPROVED** [review]
Fixed some minor style issues in mug's deployments, and package-lock issues that were making some deployments failed when installing SWC and it now looks good!

The status report says that some deployments are failing but that's fake news.

**Pull request merged**